### ISSUE TITLE
Bug 59405609: [React Native Gallery Preview]: On hitting right arrow key from 'A simple Touchable Without Feedback' text, Narrator is announcing incorrect information.

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#701]
Build Details:
OS Version: 24H2 
OS Build: 26100.6584
App: React Native Gallery Preview
App Version: 1. 0.18.0
AT Used: Narrator

Repro-Steps:
Launch the React Native Gallery-Preview app.
Navigate to the 'Expand menu' button and invoke it.
Navigate to ‘All samples’ button present in left navigation pane and invoke it.
Navigate to ‘Touchable Without Feedback’ button present under “Legacy” group and invoke it.
Navigate to  'A simple Touchable Without Feedback' text using arrow keys in scan on mode
Now hit right arrow key from above mentioned text.
Now listen to the Narrator announcement
Actual Result:
 On hitting right arrow key from 'A simple Touchable Without Feedback' text, Narrator focus is landing on 'Touchable Without Feedback' edit field and announcing incorrect information as" 
<say-as interpret-as="spell">s</say-as>, button<say-as interpret-as="spell">i</say-as><say-as interpret-as="spell">m</say-as><say-as interpret-as="spell">p</say-as><say-as interpret-as="spell">l</say-as><say-as interpret-as="spell">e</say-as>space"

Note: 
Issue is observed with left arrow key navigation in scan on mode
Same issue is observed for all the edit field available in 'Touchable Without Feedback' page
Expected Result:
On hitting right arrow key from 'A simple Touchable Without Feedback' text, Narrator should announce the first character of the 'Touchable Without Feedback' edit filed and on further hitting right arrow key, it should announce the next characters information accordingly.

User Impact:
Users who rely on Screen reader will face difficulties, if Narrator is announcing incorrect information while navigating  to the edit fields using left/right arrow keys in scan on mode

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”
